### PR TITLE
feat: change finished step layout

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- For the Finished and Cancellation failed step, this PR makes some changes.
- It moves the top image to the bottom and a separate 'share this win/tip' for better visiblity
- It does not display the ETHflow stepper for these two states

Screenshot 2024-09-12 at 13 13 20
Screenshot 2024-09-12 at 13 13 13

Screenshot 2024-09-12 at 13 44 39

### CODE REVIEW

**Review by elena-zh — COMMENTED** [review]
Changes look good, however, I'd make the top section more compact as even on a big screen (22") the modal exceeds the screen's height and requires a scroll bar to see the 'Share' button image

Also, when 'Traded' modal pops-up, it also contains a scrollbar
image
image
image

In a mobile view the bottom section is not visible and can be easily closed  (using the top arrow) without noticing it
image
image
image

**Comment by fairlighteth** [comment]
Added confetti for finish step:

https://github.com/user-attachments/assets/922784ad-4a28-42e2-9cb8-227018883485

**Review by elena-zh — APPROVED** [review]
Looks a bit better, thank you for addressing this!
Still there is a scrollbar on the popped-up modal, but lets see how it goes with user testing :) 
image

**Comment by fairlighteth** [comment]
@elena-zh yes this scrolling is expected, given this opens as a modal

**Comment by elena-zh** [comment]
@fairlighteth , as discussed in the [thread](https://cowservices.slack.com/archives/C0361CDG8GP/p1726145398412009), it would be nice to show confetti for the Surplus case only, not for every trade. 
Also, the confetti look a bit slow. See the video: https://www.loom.com/share/6feb9099b469403bb4d541feaab5aefc

**Comment by fairlighteth** [comment]
@elena-zh this has now been addressed

**Comment by elena-zh** [comment]
Hey @fairlighteth , the issue with a slow confetti is still here. 
**To reproduce:** 

1. place an order on a desktop version, select a high slippage amount
2. Close the progress bar
3. Wait till the order is executed --> surplus modal will pop-up, the confetti effect will be slower than needed. 

Also just to let you know, the confetti effect seems to be useless in a mobile device:  when an order is executed, the notification pops-up. It covers the whole screen, and the confetti effect becomes hidden under it.

**Comment by fairlighteth** [comment]
@elena-zh I tried to replicate the performance issue, but I couldn't. I did read something about if your device is in 'energy saving' mode. You think that is the case? Either way I think we should open a separate issue for this as we might need to consider changing the confetti library.

As for mobile, when you close the progress bar, it still shows up as a modal with the confetti fully visible. So I'd keep it.

**Review by elena-zh — APPROVED** [review]
@fairlighteth , my device is not in the energy-saving mode, and a Fortune cookie confetti work fine for me in this PR. 
But anyways, in order not to block the PR, approving it :) Opened #4884 bug for the slow-confetti issue.

File: apps/cowswap-frontend/src/common/pure/TransactionSubmittedContent/index.tsx [hunk 1]
```diff
@@ -99,7 +102,7 @@ export function TransactionSubmittedContent({
         <>
           {!isProgressBarSetup && <styledEl.Title>{getTitleStatus(activityDerivedState)}</styledEl.Title>}
           {showSafeSigningInfo && <GnosisSafeTxDetails chainId={chainId} activityDerivedState={activityDerivedState} />}
-          <EthFlowStepper order={order} showProgressBar={!!showProgressBar && !cancellationFailed} />
+          {!cancellationFailed && !isFinished && <EthFlowStepper order={order} showProgressBar={!!showProgressBar} />}
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/common/pure/TransactionSubmittedContent/index.tsx:105]: `cancellationFailed` is included in the `isFinished` boolean

```suggestion
          {!isFinished && <EthFlowStepper order={order} showProgressBar={!!showProgressBar} />}
```

**Review by alfetopito — APPROVED** [review]

File: apps/cowswap-frontend/src/common/pure/TransactionSubmittedContent/index.tsx (hunk 1 above)
Comment (fairlighteth) [reply apps/cowswap-frontend/src/common/pure/TransactionSubmittedContent/index.tsx]: Thank you!

**Review by shoom3301 — APPROVED** [review]

**Pull request merged**